### PR TITLE
Initial move of box2d related files

### PR DESCRIPTION
**Pull request by spydon**
Examples are coming in the next PR.

### CODE REVIEW

**Comment by erickzanardo** [comment]
LGTM, only two things I think is missing, adding the `.github` folder, with the CI config (you can just copy it from `flame_tiled`, or any other project we have), and a more meaningful changelog message on the CHANGELOG file

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## [0.0.1] - TODO: Add release date
```
Comment (erickzanardo) [inline CHANGELOG.md]: I think can just drop this TODO

Comment (spydon) [reply CHANGELOG.md]: Formatted it like the one on the main repo

**Review by erickzanardo — APPROVED** [review]

**Comment by spydon** [comment]
The linting won't be green until we remove the viewport from flame.

**Comment by erickzanardo** [comment]
> The linting won't be green until we remove the viewport from flame.

Shouldn't that class be moved here and the import changed to use the local file?

**Comment by spydon** [comment]
> > The linting won't be green until we remove the viewport from flame.
> 
> Shouldn't that class be moved here and the import changed to use the local file?

Aah, of course. Thought I had done that.
Still some problems, temporarily fixed by renaming the import.

**Pull request merged**